docs/design/handle_support.md

```diff
@@ -38,15 +38,15 @@ As handles are revocable they provide strong privacy by default. If a user feels
 		a) If the handle is selected and the same handle is mapped to DIFFERENT user then fail the `update_identity` request.  
 		b) If the handle is selected and the same handle is mapped to SAME user then do nothing.  
 		c) If the handle is selected and the handle does NOT EXIST in `mosip_idrepo.handle` table, create entry in handle table.  
-		d) If there are any unselected handles (when compared with stored selectedHandles and input selectedHandles), mark the handles as deleted in the `mosip_idrepo.handle` table.  
-	Note: Add `is_active` and `is_deleted` columns in the `mosip_idrepo.handle` table.
+		d) If there are any unselected handles (when compared with stored selectedHandles and input selectedHandles), mark the handle status as `DELETE` in the `mosip_idrepo.handle` table.  
+	Note: Add `status` column in the `mosip_idrepo.handle` table.
 	6. Update identity data (demo, docs & bio) in the uin table.
 
 	In CredentialServiceManager, when we fetch the list of handles for a UIN to issue credentials:
-	1. Issue credential event to be raised if the record is active and not deleted.
-	2. If the record is deleted, then send `REMOVE_ID` event to IDA.  
+	1. Issue credential event to be raised if the handle status is `ACTIVATED`.
+	2. If the handle status is `DELETE`, then send `REMOVE_ID` event to IDA and mark the handle status as `DELETE_REQUESTED`.  
```

Comment:
status of handles should be updated as "DELETE_REQUESTED" when a deactivated UIN is picked by the job.

Reply:
duplicate comment- https://github.com/mosip/id-repository/pull/955#discussion_r1613374193